eng/Versions.props

```diff
@@ -27,7 +27,7 @@
   <!-- Get .NET Framework reference assemblies from NuGet packages -->
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
-    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-004</MicrosoftNetFrameworkReferenceAssembliesVersion>
+    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-5</MicrosoftNetFrameworkReferenceAssembliesVersion>
```

Comment:
Is this really latest? Why the change in versioning format?

Reply:
It is the latest version we produced so far. I've discussed versioning with @dsplaisted offline before producing the packages and this was deemed a non-issue. Could we run into any issues with this versioning format change?

Reply:
Looking at https://semver.org/, I think the correct formatting would be `1.0.0-alpha.5` to make sure the packages are sorted correctly if we're not padding with zeroes.

We don't need to fix this before updating to use the new packages though.